spec/spec_helper.rb

```diff
@@ -1,3 +1,3 @@
-ENV['RS_SETFILE'] ||= 'centos8-64'
+ENV['RS_SETFILE'] ||= 'redhat8-64'
```

Comment:
redhat is not rocky? for vox pupuli, we switched from centos8 to centos stream 8, would that work for you as well?

Reply:
I had tried both rocky8 and redhat8, but neither one came up as something that docker could provision for the github actions.

We have an old internal pipeline that is testing beaker-pe against beaker-rspec, and the RS_SETFILE value ends up being consumed by beaker-hostgenerator and would need to produce something found in our vmpooler instance. We no longer have centos-8 vms there. It's not clear to us where testing beaker-rspec with newer beaker-pe commits is still worthwhile though.